content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -97,6 +99,18 @@ selected:
    However, when creating a control point specifically for quality control purposes, using a
    different check type is probably more effective.
 
+.. note::
+   When creating a |QCP| with the :guilabel:`Worksheet` or :guilabel:`Spreadsheet` check types, it
+   is necessary to specify a *Quality Worksheet Template* or *Quality Spreadsheet Template* in the
+   :guilabel:`Template` field that appears below the :guilabel:`Type` field.
+
+   The selected template is duplicated for each quality check created by the |QCP|, and must be
+   filled out to complete the quality check.
+
+   To create a new Quality Worksheet/Spreadsheet Template, open the :menuselection:`Quality` app,
+   navigate to :menuselection:`Configuration -> Quality Worksheet/Spreadsheet Templates`, and click
+   :guilabel:`New`.
```

Comment:
Friendly suggestion: Could consider rewording for brevity since "Quality Worksheet/Spreadsheet" is already mentioned earlier. Referring to it as "template" might be sufficient, especially given the detailed instructions that follow. Additionally, capitalization of "quality worksheet/spreadsheet" outside of a navigational instruction might not be necessary here.

```suggestion
   To create a new template, navigate to :menuselection:`Quality app --> Configuration -> Quality Worksheet/Spreadsheet Templates`, and click
   :guilabel:`New`.
```
Let me know if you intentionally took out "Quality app" out of the menuselection directive, or prefer the current phrasing for any reason! Just trying to be helpful in reviews (: